contrib/endpoints/src/api_manager/config_test.cc

```diff
@@ -93,13 +93,13 @@ TEST(Config, ServerConfigProto) {
 }
 
 static const char kInvalidServerConfig[] = R"(
-service_control_config {
-  type: 1
-  config {
-    cache_entries: 1020
-    flush_interval_ms: 15
+  service_control_config {
+    type: 1
```

Comment:
Actually, the dev guide only shows the advantage of using raw strings. There is no rules on indentation and here is why: various parsers can treat whitespaces differently. Some may ignore them and some might use them as delimiters. Here is a hint about this in the document:
"
Note that indentation rules, combined with the fact that raw string literals may contain newlines, leave you with an awkward choice on how to indent the first line of your raw string block. Because text protobufs ignore whitespace, this problem can be avoided by throwing in a leading newline (ignored by the parser) in that case, but not all uses of raw strings are so forgiving."

Here is the document link:

https://g3doc.corp.google.com/devtools/library_club/g3doc/totw/64.md?cl=head